rocketpy/environment/environment.py

```diff
@@ -2365,7 +2365,7 @@ def process_forecast_reanalysis(self, file, dictionary):
         self.wind_vs = wind_vs
         self.levels = levels
         self.temperatures = temperatures
-        self.time_array = time_array
+        self.time_array = time_array[:].tolist()
```

Comment:
I think we should keep consistency here. Please notice that all the variables in this block are being saved as as numpy masked arrays (not netcdf4 objects), changing the type of just one is not a good idea imo. 

I think that what we should do is to modify the `get_instance_attributes`method to automatically convert numpy array objects to lists before saving it. Some classes even have a "to_serializable" object. If you find this complicated, you should definitely us try/except block inside the `get_instance_attributes` function.

```suggestion
        self.time_array = time_array
```

Reply:
I am a little confused here.

All the variables in this block are saved as either lists or masked arrays, the only exception was `time_array` which was incorrectly saved as netcdf4 object

Other parts of the code that also define this attribute already did this exact conversion (using `.tolist()`). I just made these two cases consistent with the rest

Reply:
As discussed, you're right here, as usual